packages/big-design/src/components/Input/Input.tsx

```diff
@@ -147,19 +147,9 @@ class StyleableInput extends React.PureComponent<InputProps & PrivateProps, Inpu
   }
 
   private renderChips() {
-    const { chips, onChipDelete } = this.props;
+    const { chips } = this.props;
 
-    if (!chips) {
-      return null;
-    }
-
-    return chips.map((chip, key) =>
-      onChipDelete ? (
-        <Chip key={key} label={chip} marginBottom="none" onDelete={onChipDelete(chip)} />
-      ) : (
-        <Chip key={key} label={chip} marginBottom="none" marginTop="none" />
-      ),
-    );
+    return chips && chips.map((chip, key) => <Chip key={key} marginBottom="none" {...chip} />);
```

Comment:
Can we use something like `chip.label` as the `key`? In theory there shouldn't be any duplicate labels.

Reply:
Sure.